src/py42/modules/alertrules.py

```diff
@@ -0,0 +1,38 @@
+class AlertModule(object):
+    def __init__(self, microservice_client_factory, user_client):
+        self.microservice_client_factory = microservice_client_factory
+        self._user_client = user_client
+        self._alert_client = None
+
+    @property
+    def rules(self):
+        return self.microservice_client_factory.get_alert_rules_client(
+            self.microservice_client_factory.get_detection_list_user_client(self._user_client)
+        )
+
+    def get_all(self, sort_key="CreatedAt", sort_direction="DESC"):
```

Comment:
If  `AlertRulesModule` contains `AlertClient`, it will allow excess to other methods  as well, that existed earlier. Is that the expectation? 
 Do we want to support access to only `get_all` and `get_rules_by_name` or all?

Reply:
I think the idea is that they are private, so you only get access to the methods off that client that you need.